packages/terra-form-select/src/_Frame.jsx

```diff
@@ -374,6 +390,15 @@ class Frame extends React.Component {
       customProps.className,
     ]);
 
+    const { searchValue } = this.state;
+
+    let ariaLiveText = '';
+    const children = MenuUtil.filter(options, searchValue, optionFilter);
```

Comment:
This might be worth looking into a little more. This will cause a double filter each render for searchable variants. The first filter will happen here, the second filter will happen in the Menu itself.

Reply:
I did see both filters here, and the first thing I was thinking about is that if the list isn't that large, I don't think double filtering is a large concern. The trick to aria-live is that if it's not already mounted on the screen before it's used, then it won't send the user's results to a screen reader.

Reply:
I'll see what it takes to do the filtering at the frame level, that pass it on through to the dropdown.

Reply:
The `Frame` was originally designed so that it would be possible for users to create a custom wrapper and render content into the dropdown. Similar to the implementation in this package where the `Select.jsx` file is a wrapper around `Frame.jsx`. The `Select` uses the `DropdownMenu` as the content.

I am not aware of any consumers using the Frame this way currently and discussions may lead to suggest they shouldn't anyway. But if we move any of the Menu logic into Frame it will break the generic API the Frame was built on top of. 

It would be worth talking through, if we want to abandon the API of the Frame it should be dissolved. The `Select.jsx` and `Frame.jsx` would merge into a single file.

Reply:
Here's one option we can do.

When we mount the select box, we can create a new node that is independent of react, and append that to the dom. We can give it visually hidden styles so that it does not display at all on the screen. From there, when we do the no results filter in the menu, we can use vanillaJS to update the results of the live region with the new text. That will cause the screen reader to update results

Reply:
There is also the consideration of making an aria-live region for terra-base, and provide an api through terra base that allows for developers to communicate results like that to a screen reader

Reply:
I like the idea of creating a new node that is independent of react, and append that to the dom when we mount the select box for the aria-live text. I also think we should look into corporating that into terra-base as it is a good candidate for handling global concerns which I think aria-live makes sense to handle at a global app level.